tests/test_cursor.py

```diff
@@ -71,6 +71,29 @@ def test_iterator(self, cursor):
         self.assertEqual(list(cursor), [(1,)])
         self.assertRaises(StopIteration, cursor.__next__)
 
+    @with_cursor
+    def test_cache_size(self, cursor):
+        # To test cacheing, we need to make sure the query is unique, otherwise
+        # we might accidentally pick up the cache results from another CI run.
+        unique_comment = "-- " + str(datetime.utcnow())
```

Comment:
I thought it would be better to define the query variable as follows.
query = `SELECT * FROM one_row -- {0}`.format(str(datetime.utcnow()))